<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Lukáš Tinkl &lt;ltinkl@redhat.com&gt;</author>
 <class>WifiSecurity</class>
 <widget class="QWidget" name="WifiSecurity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wifi Security</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QComboBox" name="securityCombo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>WEP key or passphrase</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>LEAP</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dynamic WEP (802.1x)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>WPA &amp; WPA2 Personal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>WPA &amp; WPA2 Enterprise</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>S&amp;ecurity:</string>
     </property>
     <property name="buddy">
      <cstring>securityCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="securityNone"/>
     <widget class="QWidget" name="securityWep">
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&amp;Key:</string>
         </property>
         <property name="buddy">
          <cstring>wepKey</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="KLineEdit" name="wepKey">
         <property name="maxLength">
          <number>26</number>
         </property>
         <property name="passwordMode">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="cbShowWepKey">
         <property name="text">
          <string>&amp;Show key</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>WEP index&amp;:</string>
         </property>
         <property name="buddy">
          <cstring>wepIndex</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="wepIndex">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>1 (default)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Au&amp;thentication:</string>
         </property>
         <property name="buddy">
          <cstring>wepAuth</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="wepAuth">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>Open System</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Shared Key</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="securityLeap">
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>&amp;Username:</string>
         </property>
         <property name="buddy">
          <cstring>leapUsername</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="leapUsername"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>&amp;Password:</string>
         </property>
         <property name="buddy">
          <cstring>leapPassword</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="KLineEdit" name="leapPassword">
         <property name="passwordMode">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="cbShowLeapPassword">
         <property name="text">
          <string>&amp;Show password</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="securityWpaPersonal">
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>&amp;Password:</string>
         </property>
         <property name="buddy">
          <cstring>psk</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="KLineEdit" name="psk">
         <property name="passwordMode">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="cbShowPsk">
         <property name="text">
          <string>&amp;Show password</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>stackedWidget</sender>
   <signal>currentChanged(int)</signal>
   <receiver>securityCombo</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>securityCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>85</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
